### DISCUSSION TITLE
Copy commit hash on mobile

### DISCUSSION THREAD

**Discussion by mback2k** [community/community#3538] [Mobile]
It would be great if it were possible to copy the commit hash on mobile while viewing commits in a PR or on a branch.

With this feature those hashes could be used as reference in issues and other apps.

**Comment by autumndavis248999** [comment]
****

**Comment by aconsuegra** [comment]
Hi @mback2k, thanks for the feedback!

Are you using the iOS app? The feature to copy the commit hash is available in our Android app, in the commit details screen.

We hope to add the same functionality to iOS soon!

**Reply by mback2k** [reply]
Yep, I am on iOS. I am looking forward to that!
